exercises/simple-linked-list/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+pub struct SimpleLinkedList<T> {
+    // Delete this field
+    // _dummy is needed to avoid unused parameter error during compilation
+    _dummy: T
+    // head
+    // size
+}
+
+// Implementation of basic functions
+// Use SimpleLinkedList as Stack (LIFO - Last In, First Out)
```

Comment:
`impl<T> SimpleLinkedList<T>` will always be an implementation of basic functions for `SimpleLinkedList<T>`; we don't need that comment. The tests should ensure that it has stack semantics, so we can also eliminate that comment. 

The goal here is to maximize the degree to which students get to figure things out for themselves without prompting.